go/Makefile

```diff
@@ -26,4 +27,8 @@ verify-fmt:
 vet:
 	go vet ./...
 
-.PHONY: build clean test run-examples fmt verify-fmt vet
+functest:
+	go test ./tests/... -c -o ./tests/tests.test
+	./tests/tests.test
```

Comment:
You can also invoke it via `go test`, but then you don't get the nice testresult output from ginkgo.

Reply:
Switchte to pure `go test`.

Reply:
Please don't just go ahead and make changes just because I bring up a topic! I'm clearly quite inexperienced in this area, so a lot of the time I'm pointing out something that looks funny to me but is actually correct... That's just an opportunity to learn :)

If going through the extra step gives us better output (which I just tried and indeed seems to be the case) then it's worth doing so! Just find a better name for the output binary... What about 'tests/suite', for example?

Or we can even merge it like this and change it later if we come up with a better name :)

Reply:
Named it now `tests/functest.bin`.